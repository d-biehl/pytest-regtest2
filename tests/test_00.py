import pdb
import pytest

def test0(regtest):
    print >> regtest, range(10)
