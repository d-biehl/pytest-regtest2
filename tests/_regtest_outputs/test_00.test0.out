this is the expected output0x09
